packages/x-data-grid-pro/src/tests/rows.DataGridPro.test.tsx

```diff
@@ -398,12 +398,12 @@ describe('<DataGridPro /> - Rows', () => {
           await vi.advanceTimersByTimeAsync(10);
         });
         expect(getColumnValues(0)).to.deep.equal(['Nike', 'Adidas', 'Puma']);
-        expect(vi.getTimerCount()).to.equal(3);
+        expect(vi.getTimerCount()).to.equal(2);
 
         await act(async () => {
           await vi.advanceTimersByTimeAsync(100);
         });
-        expect(vi.getTimerCount()).to.equal(1);
+        expect(vi.getTimerCount()).to.equal(0);
```

Comment:
Reversal of https://github.com/mui/mui-x/pull/18275/files#diff-d5273702b71b80e2e681949d9c9c03bf5b6edd0d383a7c30cdd78993b49f9598

Reply:
I must say I don't understand how `useStoreEffect` affected this, it doesn't use any timer directly.